### PR TITLE
Added greek translation (el_GR.lang)

### PR DESCRIPTION
**Pull request by ttofis**
Added greek translation. Some stuff in greek we use (and there is no real greek word) the english word. Some stuff are content to change (complicatednessof the language) but for now this is pretty good.

### CODE REVIEW

File: src/main/resources/assets/jei/lang/el_GR.lang [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+jei.tooltip.liquid.amount=%,d mB
+jei.tooltip.transfer=Μετακίνησε αντικείμενα
+jei.tooltip.recipe.ore.dict=Δέχεται: %s
+jei.tooltip.item.colors=Χρώματα: %s
+jei.tooltip.shapeless.recipe=Άμορφη Συνταγή
+jei.tooltip.cheat.mode=Λειτουργία Cheat
+
+# Error Tooltips
+jei.tooltip.error.recipe.transfer.missing=Ελλιπές Αντικείμενα
+jei.tooltip.error.recipe.transfer.inventory.full=Το inventory σου είναι υπερβολικά γεμάτο
```
Comment (ttofis) [inline src/main/resources/assets/jei/lang/el_GR.lang:16]: No real word for inventory

File: src/main/resources/assets/jei/lang/el_GR.lang [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+gui.jei.category.fuel.smeltCount=Ψήνει %s αντικείμενα
+gui.jei.category.fuel.burnTime=Ώρα Ψησίματος: %s
+gui.jei.category.brewing=Ζύμωση
+gui.jei.category.brewing.steps=Βήματα: %s
+gui.jei.category.itemDescription=Περιγραφή
+
+# DEBUG (for debug mode, do not need translation)
+description.jei.wooden.door.1=Wooden Doors allow you to block monsters from entering your building.\nTesting sentences.
+description.jei.wooden.door.2=Clicking on a door changes its state from open to closed and visa versa.
+description.jei.wooden.door.3=Wooden Doors can be opened/closed via Redstone Circuits.
```
Comment (ttofis) [inline src/main/resources/assets/jei/lang/el_GR.lang:102]: Did not translate as you pointed out.

**Review by ttofis — COMMENTED** [review]
Added some comments to solve any questions that you might have if you take a look at the translation.

**Comment by mezz** [comment]
Thank you!

**Pull request merged**